code/modules/food_and_drinks/drinks/drinks/cans.dm

```diff
@@ -1,18 +1,30 @@
 /obj/item/reagent_containers/food/drinks/cans
-	var/canopened = 0
+	var/canopened = FALSE
 	var/is_glass = 0
 	var/is_plastic = 0
+	var/shaken = 0
```

Comment:
minor nitpick: `times_shaken` would be more descriptive here as shaken reads as if it's a boolean value